commands/drive/teleop/swerve/XboxSwerveDrive.java

```diff
@@ -35,7 +38,7 @@ public XboxSwerveDrive(SwerveDrive swerveDriveBase, MustangController controller
 
         MAX_VELOCITY = swerveDriveBase.getMaxVelocityMetersPerSecond();
         MAX_ANGULAR_VELOCITY = swerveDriveBase.getMaxAngularVelocityMetersPerSecond();
-
+        SmartDashboard.putNumber("Swerve Speed", 0.66);
```

Comment:
why is this added as part of this change? and also, what does it mean? 0.66?

Reply:
bump

Reply:
so sebby wanted to slow down the swerve during drive practice one day so we added this and the comment below to allow for variable speed. I removed  it